multi_partner_learning.py

```diff
@@ -504,25 +694,33 @@ def collaborative_round_fit(model_to_fit, train_data, val_data, batch_size):
 
         return history
 
-    def log_collaborative_round_partner_result(self, partner, partner_index, validation_score):
+    def log_collaborative_round_partner_result(
+        self, partner, partner_index, validation_score
+    ):
         """Print the validation accuracy of the collaborative round"""
 
         epoch_nb_str = f"Epoch {str(self.epoch_index).zfill(2)}/{str(self.epoch_count - 1).zfill(2)}"
         mb_nb_str = f"Minibatch {str(self.minibatch_index).zfill(2)}/{str(self.minibatch_count - 1).zfill(2)}"
         partner_id_str = f"Partner id #{partner.id} ({partner_index}/{self.partners_count - 1})"
         val_acc_str = f"{round(validation_score, 2)}"
 
-        logger.debug(f"{epoch_nb_str} > {mb_nb_str} > {partner_id_str} > val_acc: {val_acc_str}")
+        logger.debug(
+            f"{epoch_nb_str} > {mb_nb_str} > {partner_id_str} > val_acc: {val_acc_str}"
+        )
 
     def update_iterative_results(self, partner_index, fit_history):
         """Update the results arrays with results from the collaboration round"""
 
         validation_score = fit_history.history["val_accuracy"][0]
 
-        self.scores_last_learning_round[partner_index] = validation_score  # TO DO check if coherent
+        self.scores_last_learning_round[
+            partner_index
+        ] = validation_score  # TO DO check if coherent
```

Comment:
Is it possible the rearrange this definition as `validation_score = (...)`?

Reply:
What do you mean by rearranging?

Reply:
I might have glitched on this one, my bad!